pwndbg/color/__init__.py

```diff
@@ -153,20 +161,20 @@ def __getattr__(self, attr):
         raise AttributeError(f"ColorConfig object for {self._namespace} has no attribute '{attr}'")
 
 
-def generateColorFunction(config: str):
-    # the `x` here may be a config Parameter object
+def generateColorFunction(config: str, _globals=globals()):
+    # the `config` here may be a config Parameter object
     # and if we run with disable_colors or if the config value
     # is empty, we need to ensure we cast it to string
     # so it can be properly formatted e.g. with:
     # "{config_param:5}".format(config_param=some_config_parameter)
-    function = lambda x: str(x)
```

Comment:
I really need to get `pylint` working again, it can catch things like this: https://pylint.pycqa.org/en/latest/user_guide/messages/warning/unnecessary-lambda.html